src/style.scss

```diff
@@ -0,0 +1,10 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './veriables';
+@import './styles/blocks/main';
+@import './styles/blocks/contentPhoto';
+@import './styles/blocks/contentText';
+@import './styles/blocks/contentCode';
+@import './styles/index';
+@import './styles/blocks/reviws';
```

Comment:
There is a typo in the import path: 'reviws' should be 'reviews'. Ensure the file name matches the actual file.